test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -10123,18 +10140,28 @@ def __init__(self, hostname=None, server=None, pbsconf_file=None,
         rg = self.parse_resource_group(self.hostname, self.resource_group_file)
         self.resource_group = rg
 
+        if id is None:
+            sc_name = self.server.dflt_sched_name
+        else:
+            sc_name = id
+        self.db_access = db_access
+        self.version = None
```

Comment:
.. Here the git diff is not showing properly. 

Existing code. Maybe i can add the version.